.github/workflows/supply-chain.yml

```diff
@@ -0,0 +1,34 @@
+name: supply-chain
+
+on:
+  push:
+    branches:
+     - main
+     - release/**
```

Comment:
If the branches listed here disagree with [this same file on `main`](https://github.com/divviup/libprio-rs/blob/6449bbab899d2025e218cc954fdaa6eafed12cad/.github/workflows/supply-chain.yml#L6), then which one wins? I guess we can take this change, and if we see that the `supply-chain` jobs don't get run on PRs into `release/**` branches, then we can update the `branches:` specification in `supply-chain.yaml` on `main`.

Reply:
The new supply-chain workflow did run on this PR, so it appears that the target branch wins for PR triggers. I plan to update main in a follow-up PR anyway for the sake of the next time we create a release branch.